store/tikv/metrics.go

```diff
@@ -172,6 +172,31 @@ var (
 			Name:      "load_safepoint_total",
 			Help:      "Counter of load safepoint.",
 		}, []string{"type"})
+
+	secondaryLockCleanupWorkerCounter = prometheus.NewGauge(
```

Comment:
How about naming it _xxxGauge_ instead of _xxxCounter_?

Reply:
ok